.github/workflows/appimage.yml

```diff
@@ -1,8 +1,20 @@
-name: Build AppImage SLAM Server
+name: Build AppImage Cartographer
 
 on:
   workflow_dispatch:
+    inputs:
+      release_type:
+        required: true
+        description: Release version for this appimage (latest/rc/stable)
+        type: string
+        default: latest
   workflow_call:
+    inputs:
+      release_type:
+        required: true
+        description: Release version for this appimage (latest/rc/stable)
+        type: string
+        default: latest
```

Comment:
Ah! Here, the description and default are NOT needed, since, as you noted above, it's required, and will only ever be coming from other scripts.